### PR TITLE
Design checlist everywhere

### PR DESCRIPTION
**Pull request by mnaumanali94**

### CODE REVIEW

**Comment by lottamus** [comment]
:heavy_check_mark: Deploy preview for *stoplightio* ready!


:hammer: Explore the source changes: a53144452116cf5e02dbefecdbb9d07a54ce0157

:mag: Inspect the deploy logs: [https://app.netlify.com/sites/stoplightio/deploys/5fff85fbf77c700008d0fbf2](https://app.netlify.com/sites/stoplightio/deploys/5fff85fbf77c700008d0fbf2?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-1722--stoplightio.netlify.app](https://deploy-preview-1722--stoplightio.netlify.app?utm_source=github&utm_campaign=bot_dp)

File: netlify/subpages/api-development-guide.md [hunk 1]
```diff
@@ -107,6 +107,8 @@ specifications such as OpenAPI V2 and V3 (V2 formerly known as Swagger v2.0), ne
 machine-readable format have rapidly advanced. API planning and design tools continue to mature and now easily support
 open source solutions for graphical modeling, documentation, validation, and mocking, all from your API schema.
 
+<a href="/api-design-checklist-download?utm_source=blog&utm_medium=inline-cta&utm_campaign=design-checklist">![The Stoplight Platform](/images/cta-buttons/blog-cta-design-checklist.svg)</a>
```
Comment (paulatulis) [inline netlify/subpages/api-development-guide.md:110]: I think change this to` utm_source=guide` right?

File: netlify/subpages/api-development-guide.md [hunk 2]
```diff
@@ -257,6 +259,8 @@ In either case, you can connect your mock servers to the latest OpenAPI document
 in version control, you can automatically generate new servers at a temporary subdomain. Collaborators can then access
 the mock API via the exact same path and receive realistic API responses.
 
+<a href="/api-design-checklist-download?utm_source=blog&utm_medium=inline-cta&utm_campaign=design-checklist">![The Stoplight Platform](/images/cta-buttons/blog-cta-design-checklist.svg)</a>
```
Comment (paulatulis) [inline netlify/subpages/api-development-guide.md:262]: Same here

File: netlify/subpages/api-types.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ They may require registration, and use of an API key, or may be completely open.
 [Food Hygiene Rating API](https://api.ratings.food.gov.uk/help) provided by the UK government. Any developer can access
 it, without even registering, allowing app builders to include governmental data on restaurant standards in their apps.
 
-<a href="/api-design-checklist-download?utm_source=blog&utm_medium=inline-cta&utm_campaign=design-checklist">![The Stoplight Platform](/images/cta-buttons/blog-cta-design-checklist.svg)</a>
+<a href="/api-design-checklist-download?utm_source=guide&utm_medium=inline-cta&utm_campaign=design-checklist">![The Stoplight Platform](/images/cta-buttons/blog-cta-design-checklist.svg)</a>
```
Comment (paulatulis) [inline netlify/subpages/api-types.md:79]: Not sure if you want to have a separate utm for these seo pages vs. guides, but pointing out in case you do.

**Review by paulatulis — APPROVED** [review]

**Pull request merged**